src/display/touch_manager.js

```diff
@@ -78,26 +80,61 @@ class TouchManager {
   }
 
   #onTouchStart(evt) {
-    if (this.#isPinchingDisabled?.() || evt.touches.length < 2) {
+    if (this.#isPinchingDisabled?.()) {
+      return;
+    }
+
+    if (evt.touches.length === 1) {
+      const pointerDownAC = (this.#pointerDownAC = new AbortController());
```

Comment:
Possibly I'm not fully understanding the code, but is there ever a risk that we could be "overwriting" an existing `this.#pointerDownAC` here?
If so, is that not an issue since it'd lead to *duplicate* event listeners being attached?

Would the following make sense?
```suggestion
    if (evt.touches.length === 1 && !this.#pointerDownAC) {
      const pointerDownAC = (this.#pointerDownAC = new AbortController());
```